client/lib/push-notifications/index.js

```diff
@@ -0,0 +1,141 @@
+/**
+ * External dependencies
+ */
+var Emitter = require( 'lib/mixins/emitter' ),
+	debug = require( 'debug' )( 'calypso:push-notifications' );
+
+var initializeState,
+	registerServiceWorker;
+
+initializeState = function() {
```

Comment:
Tiny thing: This is more concisely written as:
`function initializeState() {` -- & you can remove the `var` definition above & the semicolon after the closing bracket.

Same with `registerServiceWorker`